CHANGELOG.md

```diff
@@ -1,15 +1,17 @@
 # Changelog
 
-## 4.1.1
+## 4.2.0
 
 - **Feature:**
   - Add performance improvements when creating transactions and processing L1 blocks
+  - L5 Binance
```

Comment:
I know a few other things changed. Other than making sure this changelog is good with all of the features/fixes, this PR is good